test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatTestUtils.cs

```diff
@@ -212,28 +212,55 @@ public static XmlReaderSettings CreateSafeSettings(bool ignoreWhiteSpace = false
         public static PackageReferenceArgs GetPackageReferenceArgs(string packageId, SimpleTestProjectContext project)
         {
             var logger = new TestCommandOutputLogger();
-            var packageDependency = new PackageDependency(packageId);
-            return new PackageReferenceArgs(project.ProjectPath, packageDependency, logger);
+            return new PackageReferenceArgs(project.ProjectPath, logger)
+            {
+                PackageId = packageId
+            };
+        }
+
+        public static PackageReferenceArgs GetPackageReferenceArgs(string packageId, SimpleTestProjectContext project, string packageVersion = "*",
```

Comment:
This method seems to have the same parameters as the overloaded method defined at line 244.
Either: 

- Update all your references to this method to use the method at line 244 and remove this method or,
- Make an internal call to the method at line 244 to avoid duplicate logic